theano/tensor/var.py

```diff
@@ -692,6 +692,9 @@ def cumsum(self, axis=None):
     def cumprod(self, axis=None):
         return theano.tensor.extra_ops.cumprod(self, axis)
 
+    def searchsorted(self, v, side='left', sorter=None):
```

Comment:
This isn't used in your test. Just change one test to call it by this way. This make sure we won't break/remove that interface by mistake.